forms/models.py

```diff
@@ -1,3 +1,13 @@
 from django.db import models
 
-# Create your models here.
+
+class Form(models.Model):
+    name = models.CharField(max_length=255, help_text='Name form', null=False)
+    descriptions = models.CharField(max_length=255, help_text='Description form')
```

Comment:
please name it in single form: `description`

Reply:
YES SIR